edith-step/src/App.js

```diff
@@ -1,46 +1,102 @@
 import React, {Component} from 'react';
-import ReceiptInput from './ReceiptInput';
-import LineChart, {BarChart} from './UserChart';
-import axios from 'axios';
-//import "./App.css";
+import LineChart, {BarGraph, CategoryDoughnutChart, ItemDoughnutChart} from './UserChart';
+import ReceiptInput from './components/ReceiptInput';
+import TopNavbar from './components/TopNavbar';
+import './App.css';
 
+/**
+ * Corresponds to the different chart types.
+ */
+ const chart = {
+    'LINE': 'LINE',
+    'BAR': 'BAR',
+    'DOUGHNUT':  'DOUGHNUT' 
+}
 
-/** Main webpage for the website. */
+/**
+ * Main webpage for the website.
+ */
 class App extends Component {
   constructor() {
     super();
-    this.state = { "chartType" : LineChart }; 
-    this.updateChartType = this.updateChartType.bind(this);   
-  }
+    this.state = {'chartType': LineChart, 'dateSelection': '', 'categorySelection': ''}; 
+    this.updateChartType = this.updateChartType.bind(this); 
+    this.showItemChart = this.showItemChart.bind(this);  
+  } 
 
+  /**
+   * Updates the value of chartType in state 
+   * based on the value of {@code event}.
+   * @param event - HTML element that has been selected
+   */
```

Comment:
nit: I think there should be a new line between the description and the @params

Reply:
Done.